rest/adminapitest/admin_api_test.go

```diff
@@ -3776,6 +3776,56 @@ func TestSetFunctionsWhileDbOffline(t *testing.T) {
 	resp.RequireResponse(http.StatusOK, syncFunc)
 }
 
+func TestCollectionSyncFnWithBackticks(t *testing.T) {
+	base.SetUpTestLogging(t, base.LevelInfo, base.KeyHTTP)
+
+	// Start SG with bootstrap credentials filled
+	ctx := base.TestCtx(t)
+	config := rest.BootstrapStartupConfigForTest(t)
+	sc, err := rest.SetupServerContext(ctx, &config, true)
+	require.NoError(t, err)
+	serverErr := make(chan error, 0)
+	go func() {
+		serverErr <- rest.StartServer(ctx, &config, sc)
+	}()
+	require.NoError(t, sc.WaitForRESTAPIs())
+	defer func() {
+		sc.Close(ctx)
+		require.NoError(t, <-serverErr)
+	}()
+
+	// Get a test bucket, and use it to create the database.
+	tb := base.GetTestBucket(t)
+	defer tb.Close()
+
+	// Initial DB config
+	dbConfig := `{
+    "num_index_replicas": 0,
+    "bucket": "todo",
+    "scopes": {
+      "_default": {
+         "collections": {
+            "lists": {
```

Comment:
I'm not sure how this database can be created successfully without bucket `todo`, and scope/collection `_default.lists` existing on the server. We might need to insert tb name and find out what scope/collection is available to use as well.